doc/users/python_tutorial.rst

```diff
@@ -0,0 +1,83 @@
+Python Client Tutorial
+======================
+
+.. note:: See :ref:`Python Client API <users/python_api:Python Client API>` for complete full API specification.
+
+First, install the CCF Python package:
```

Comment:
Meta: We should add a preface to this tutorial. It describes how to use this Python package to build a client, how to talk to a live CCF service with it. Need to make it clear this is _only_ a client. From there, I might still call this `the CCF Python tools package` or similar (even though the package is _named_ CCF), but that's a nit.

Reply:
Good point! Done.